manifests/metricbeat.yaml

```diff
@@ -162,15 +169,28 @@ metadata:
   labels:
     k8s-app: metricbeat
 rules:
-- apiGroups: [""] # "" indicates the core API group
+- apiGroups: [""]
   resources:
+  - nodes
   - namespaces
   - events
   - pods
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["extensions"]
+  resources:
+  - replicasets
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["apps"]
+  resources:
+  - statefulsets
+  - deployments
+  verbs: ["get", "list", "watch"]
+- apiGroups:
+  - ""
```

Comment:
Is this correct? Likes like nothing here for the array in yaml.

Reply:
yes, "" indicates the core API group

Reply:
but I agree, I wish this was just `"core"` :smile:

Reply:
Got it. Also got confused that on line 172 and here you use a different syntax

Reply:
uhm, that's a good catch, these could be merged indeed, will need to test that